src/components/MainJoinedGroup.js

```diff
@@ -295,8 +295,8 @@ function MainJoinedGroup() {
           filteredSubgroups.map((subgroup) => (
             <ListItem
               key={subgroup.subgroupId}
-              mainTitle={subgroup.name || subgroup.subgroupName}
-              subtitle={subgroup.subTitle || subgroup.subgroupName}
+              mainTitle={subgroup.subgroupName}
+              subtitle={subgroup.subgroupCaption || subgroup.subgroupName}
```

Comment:
do we want to show the subgroupName twice if there is no subtitle?

Reply:
if not, `|| subgroup.subgroupName` should be removed from this line

Reply:
If there is no subTitle should we leave it empty ?

Reply:
wouldn't that be better than just showing the title twice after each other? (which was btw the original issue that this fix is supposed to be for)